processor/transformprocessor/internal/common/functions_test.go

```diff
@@ -211,16 +211,16 @@ func Test_newFunctionCall_invalid(t *testing.T) {
 							Fields: []Field{
 								{
 									Name:   "attributes",
-									MapKey: strp("test"),
+									MapKey: testhelper.Strp("test"),
```

Comment:
Please split off this change (strp -> testhelper.Strp) into its own PR to fix the broken tests